### ISSUE TITLE
[BUG] Can't login 

### ISSUE THREAD

**Issue by CrosbyMGO** [robvdpol/RaceControl#342]
After putting my credentials and clicking login, this error appears: "The 'META' start tag on line 2 position 14 does not match the end tag of 'HEAD'. Line 4, position 3."

**Desktop (please complete the following information):**
 - Windows 10 Home Single Language 
 - 64 bit

- F1TV PRO
- ARGENTINA
- Using VPN or proxy? = NO

**Comment by woochi1** [comment]
I cant log in to.. It said 401 Unauthorized

**Comment by kleepMan** [comment]
@CrosbyMGO Can you try again? Tried to logout and the login again and it worked for me.

**Comment by CrosbyMGO** [comment]
i'm trying to login for the first time, i can't logout from anything.

**Comment by timfrankentwf** [comment]
Can you login successfully on the main website?

**Comment by robvdpol** [comment]
Which version are you using? Please send a screenshot with the error as well.

**Comment by robertvissereindhoven** [comment]
I have the same issue...

![racecontrol error](https://user-images.githubusercontent.com/50362732/159007603-be7a6345-b506-4aa0-8f59-8e39a262c043.jpg)

**Comment by robvdpol** [comment]
Version 1.8.0 is ancient, please download the latest release.

**Comment by robertvissereindhoven** [comment]
> Version 1.8.0 is ancient, please download the latest release.
Ok, thanks, It is working now!

**Comment by CrosbyMGO** [comment]
yep i managed to do the same, and now it works ! thank you all for the replies :)
